wurst/_handles/Unit.wurst

```diff
@@ -606,3 +606,15 @@ public function unit.getLocalZ() returns real
 
 public function unit.isIllusion() returns bool
 	return IsUnitIllusion(this)
+
+public function unit.toGroup(group g)
```

Comment:
These unit. functions seem to be pointless duplicates of existing functionality.
Since you need access to both unit and group, you should just use container.xxx(element)